src/main/headers/apple_fsnotifier.h

```diff
@@ -0,0 +1,61 @@
+#if defined(__APPLE__)
```

Comment:
All the new headers miss something like
```
#ifndef __INCLUDE_APPLE_FSNOTIFIER_H__
#define __INCLUDE_APPLE_FSNOTIFIER_H__
...
#endif
```

So you can include them twice and they only will be included once.

Reply:
I ended up adding `#pragma once` instead.